src/app/app.component.html

```diff
@@ -1,5 +1,11 @@
 <div class="container">
   <app-header [chuckFace]="true"></app-header>
+  <app-quote
+    quote="Test data"
+    [pending]="false"
+    [pendingText]="'Loading' | translate"
+    [error]="false"
+    [errorText]="'Error.Request' | translate"></app-quote>
```

Comment:
i think you can keep pending text and error text in the component if it never change